frontend/src/components/messages/ReplyRequestDialog.vue

```diff
@@ -109,13 +111,17 @@ export default {
       if (this.isFormComplete && this.areValidFilesUploaded) {
         try {
           this.isLoading = true
-          await this.createReply(this.assistanceRequestId)
           await DocumentService.createDocuments(this.uploadedDocuments, this.assistanceRequestId)
+          await this.createReply(this.assistanceRequestId)
           await this.updateStatusToAssigned(this.assistanceRequestId)
           await this.updateStoredAssistanceRequest(this.assistanceRequestId)
           this.$emit('reply-success-event', true) // emit success to flag showing success message
         } catch (error) {
-          console.log(`Submit processing for reply failed  - ${error}`)
+          if (error?.response?.data?.status === 422) {
+            this.setFailureAlert('Submit processing for reply failed supporting documents failed virus scan', error)
```

Comment:
This is a little verbose for the user. Maybe something like "Supporting documents not saved. Virus scan failed."